pkg/apis/pmemcsi/v1beta1/deployment_types.go

```diff
@@ -126,10 +126,9 @@ type DeploymentCondition struct {
 	// Status of the condition, one of True, False, Unknown.
 	Status corev1.ConditionStatus `json:"status"`
 	// Message human readable text that explain why this condition is in this state
-	// +optional
 	Reason string `json:"reason,omitempty"`
 	// Last time the condition was probed.
-	// +optional
```

Comment:
`+optional` had no effect, had it?

Reply:
Not sure if that had effect on this field.